src/core/Akka/Event/EventBusUnsubscriber.cs

```diff
@@ -45,140 +42,83 @@ public EventStreamUnsubscriber(EventStream eventStream, ActorSystem system, bool
             _debug = debug;
            
         }
-
-        /// <summary>
-        /// TBD
-        /// </summary>
-        /// <param name="message">TBD</param>
-        /// <returns>TBD</returns>
+        
         protected override bool Receive(object message)
         {
-            return message.Match().With<Register>(register =>
+            switch (message)
             {
-                if (_debug)
```

Comment:
The code before was very old and still used `PatternMatch`. Refactored to a `switch` statement.